site/src/theme/externalImages.ts

```diff
@@ -162,6 +162,7 @@ export const defaultParametersForBuiltinIcons = new Map<string, string>([
 	["/icon/nexus-repository.svg", "blackWithColor"],
 	["/icon/okta.svg", "monochrome"],
 	["/icon/openai.svg", "monochrome"],
+	["/icon/positron.svg", "whiteWithColor"],
```

Comment:
```suggestion
```

I would consider this icon full color. it already looks fine on dark and light background because it has its own background color.